src/coordination/coordinator_data.cpp

```diff
@@ -17,6 +17,7 @@
 
 #include <range/v3/view.hpp>
 #include <shared_mutex>
+#include "libnuraft/nuraft.hxx"
```

Comment:
I think our convention is that 3rd party libraries are included as `<lib.hpp>`

Reply:
Good catch, will change it